dev-docs/publisher-api-reference.md

```diff
@@ -16,6 +16,9 @@ This page has documentation for the public API methods of Prebid.js.
 {: .alert.alert-warning :}
 Some methods were deprecated in Prebid 1.0. [Archived pre-1.0 documentation]({{site.baseurl}}/dev-docs/publisher-api-reference-old.html) is available.
 
+{: .alert.alert-danger :}
```

Comment:
Can we move this deprecation message above the "warning" message on line 17?

Reply:
done